adding_socialmedia_links

sowmya-1313 opened the pull request:
fixes: #5
@Dishika18 

I added social media links in index.html in information section to avoid responsive conflicts .please once review this i hope this time thier will be no conflicts .if this works i try to implement in other html files.
![image](https://github.com/user-attachments/assets/df05f0c3-9552-443f-9cbf-17c514170123)

Dishika18 commented:
@sowmya-1313, this looks good now! You can add it to the other HTML files as well. Also, if possible, consider adding a hover effect to the icons for a better user experience.

sowmya-1313 commented:
Okay

sowmya-1313 commented:
@Dishika18 #57 

I added social media icons to 
- contact.html
- index.html
- topics-details.html
- topics-listing.html

- updated style.css too. But, iam unable to add icons in **add-topics.html** file because the footer in add-topics.html is in this form. 
![image](https://github.com/user-attachments/assets/1d3e98fd-e0d9-4fa7-9448-4baf503a86d4)
even if i add icons it changes the copyright part of footer so i didn't added any icons in add-topics.html file

sowmya-1313 commented:
hi @Dishika18 

I noticed that my InsightSync PR appears in the contributions section, but it’s not showing up on the SWOC leaderboard. Could you please check this?

 inform me if i misunderstoond anything .
thank you
